### PR TITLE
feat: improve wait duration ; make it confgurable

### LINKED ISSUE #9: Reduce wait period from 30s to maybe 10s
https://github.com/influxdata/circleci-helper/blob/b09750270138212ebe6830fecbe3c139a5878419/cmd/circleci-helper/internal/waitForJobs.go#L94-L96

One of our bottleneck pipelines block on this call -- in the worst case we have to wait 30 seconds longer after the upstream jobs complete.

Dropping the wait period down to 10s would be an easy change to help the bottleneck complete faster, or we could allow it to be configurable.

I saw that the [v1 API docs mention rate limiting](https://circleci.com/docs/api/#rate-limiting):

> For HTTP APIs, when a request is throttled, you will receive HTTP status code 429. If your integration requires that a throttled request is completed, then you should retry these requests after a delay, using an exponential backoff. In most cases, the HTTP 429 response code will be accompanied by the Retry-After HTTP header. When this header is present, your integration should wait for the period of time specified by the header value before retrying a request.

I am not seeing any docs on v2 API rate limiting.

### PR DESCRIPTION
**Pull request by wojciechka**
Closes #9

### CODE REVIEW

**Comment by wojciechka** [comment]
I also cleaned up some code and renamed files to have better naming convention.

File: cmd/circleci-helper/internal/wait_for_jobs.go [hunk 1]
```diff
@@ -54,7 +58,9 @@ func WaitForJobs(ctx context.Context, logger *zap.Logger, client circle.Client,
 			result = &WorkflowsSummary{}
 		}
 
-		// report all workflows - starting with
+		pendingJobCount = 0
```
Comment (GeorgeMac) [inline cmd/circleci-helper/internal/wait_for_jobs.go:61]: Take it or leave it. Could just define this variable here. Doesn't seem to be used outside the scope of this loop?
```go
var pendingJobCount int
```

**Review by GeorgeMac — APPROVED** [review]
One take it or leave it. Looks good.

File: cmd/circleci-helper/internal/wait_for_jobs.go (hunk 1 above)
Comment (wojciechka) [reply cmd/circleci-helper/internal/wait_for_jobs.go:61]: +1

**Pull request merged**